books/app.js

```diff
@@ -0,0 +1,178 @@
+'use strict';
```

Comment:
THe files in `books/` are similar; you should remove one of them.

Reply:
removed